pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -65,28 +65,49 @@ func (c *ConfigMapUnpacker) job(cmRef *corev1.ObjectReference, bundlePath string
 							},
 							VolumeMounts: []corev1.VolumeMount{
 								{
-									Name:      "copydir",
-									MountPath: "/injected",
+									Name:      "bundle", // Expected bundle content mount
+									MountPath: "/bundle",
 								},
 							},
 						},
 					},
 					InitContainers: []corev1.Container{
 						{
-							Name:    "copy-binary",
-							Image:   c.copyImage,
-							Command: []string{"/bin/cp", "/bin/opm", "/copy-dest"},
+							Name:    "tools",
+							Image:   c.opmImage,
+							Command: []string{"/bin/cp", "-Rv", "/bin/cp", "/tools/cp"}, // Copy tooling for the bundle container to use
 							VolumeMounts: []corev1.VolumeMount{
 								{
-									Name:      "copydir",
-									MountPath: "/copy-dest",
+									Name:      "tools",
+									MountPath: "/tools",
+								},
+							},
+						},
+						{
+							Name:    "pull",
+							Image:   bundlePath,
+							Command: []string{"/tools/cp", "-Rv", "/manifests", "/metadata", "/bundle"}, // Copy bundle content to its mount
```

Comment:
Hard coding `/manifests` for now. We'll follow up with a script that uses the manifest directories label to determine what to copy.